sample/manifest.yml

```diff
@@ -5,7 +5,7 @@ applications:
 #
 - name: logging-sample-app
   instances: 1
-  buildpack: sap_java_buildpack
+  buildpack: java_buildpack
```

Comment:
Did you test this? The java_buildpack lacks some dependencies.

Reply:
Yes, I tested it.